llvm/lib/DebugInfo/DWARF/DWARFContext.cpp

```diff
@@ -1004,21 +1006,47 @@ void DWARFContext::dump(
                  DObj->getAbbrevDWOSection()))
     getDebugAbbrevDWO()->dump(OS);
 
+  std::unordered_map<uint64_t, DWARFUnit *> DwoIDToDUMap;
```

Comment:
We usually use llvm::DenseMap.
https://llvm.org/docs/CodingStandards.html#c-standard-library